pep-0710.rst

```diff
@@ -171,16 +171,146 @@ from the installer's cache.
 Backwards Compatibility
 =======================
 
-Since this PEP specifies a new file in the ``.dist-info`` directory, there are
-no backwards compatibility implications to consider in the ``provenance_url.json``
-file itself. Also, this proposal does not make any changes to the
-``direct_url.json`` described in :pep:`610` and
-:ref:`its corresponding canonical PyPA spec <direct-url>`.
+Following the :ref:`packaging:recording-installed-packages` specification,
+installers may keep additional installer-specific files in the ``.dist-info``
+directory.  To make sure this PEP does not cause any backwards compatibility
+issues, there was conducted a research whether introducing the
+``provenance_url.json`` file in the ``.dist-info`` directory is feasible and
+does not cause any backwards compatibility issues in the Python packaging
+ecosystem.
```

Comment:
```suggestion
issues, a :ref:`comprehensive survey of installers and libraries <710-tool-survey>`
found no current tools that are using a similarly-named file,
or other major feasibility concerns.
```

* Reference/link and summarize the major results of the survey
* Avoid repeating the purpose of the survey (ensuring there are no backward compat issues) twice in the same sentence
* Fix/avoid grammar errors and phrasing issues to make this passage easier to read

```